app/views/miq_policy/_action_options.html.haml

```diff
@@ -68,12 +68,8 @@
       .form-group
         .col-md-8
           #action_tags_treebox.treeview-pf-hover.treeview-pf-select
```

Comment:
Yes, you told me that, probably problem with rebasing and then a had do it over again because of migration of the whole project structure, something broke down and i had to redo it again using old codes...